### ISSUE TITLE
Quest: Leading the Charge

### ISSUE THREAD

**Issue by JaRy66** [YggdrasilWotLK/Yggdrasil-WoW#61]
**Title:** Quest: Leading the Charge (ID: 13380)
https://www.wowhead.com/wotlk/quest=13380/leading-the-charge

**Issue:** Please explain what is happening.
While you take the quest, you are supposed to kill bunch of hostile npc's while in the plane, however you are supposed to get attacked by Gargoyle Ambusher, but you can't kill them before they aggro you (they evade the damage), making the player always take damage. This combined with the fact that the built in shield barely blocks damage, and the plane being very prone to getting lit on fire, this makes the quest much harder than it needs to be, because he plane is destroyed very quickly.
Gargoyles also evaded after I attacked bunch of them with one shot, but that might be an anomaly.

**Intended functionality:** You should be able to attack the Gargoyle Ambusher even before they attack you, probably also take a look at the damage that the plane gets, because it's very easy to explode.

![WoWScrnShot_080624_222352](https://github.com/user-attachments/assets/7c85ab29-8e26-40f0-8be5-10c99f6abdcd)

**Reproduction steps:** Just start the quest and go around, trying to shoot at gargoyles

**Comment by YggdrasilWotLK** [comment]
Issue 1: Evading gargoyles - Uh-oh, roughly half the gargoyles evade all my shots at them, even when they attack me! This shouldn't happen, we will look into the issue. They should be attackable before they attack the player.

Issue 2: Gargoyle damage - The damage is as intended. With full shields up the dmg is only 3-4k per hit, which would be manageable had half the gargoyles not evaded.

Issue 3: Shields. The shields work fine against Gargoyle damage, drastically reducing damage taken at 90-100 stacks and not so much at fewer stacks. Could the issue you've seen be that the shields should for example reduce damage taken from the vehicle being on fire? Either way it's only some 9k damage per 4-ish seconds (at the same time, the ship heals about 1k per sec making that relatively minor as well). 

Tldr; I think there's only one issue here, the gargoyles evading unreasonably

Re https://www.youtube.com/watch?v=QhRTxyCnFpM

**Comment by YggdrasilWotLK** [comment]
I've fixed the evading gargoyles by having them engage a chase with the player until they reach combat range re https://github.com/YggdrasilWotLK/ELUNA-Scripts/blob/main/Quest/13380.lua
